PWGDQ/Tasks/dileptonMuMu.cxx

```diff
@@ -319,19 +339,30 @@ struct DQDileptonMuMu {
   {
     fValues = new float[VarManager::kNVars];
     VarManager::SetDefaultVarNames();
+
+    fMixHandler = new MixingHandler("mixingHandler", "mixingHandler");
+    fMixHandler->Init();
+    TString mixingVarStr = fConfigMixingVariables.value;
+    SetUpMixing(fMixHandler, mixingVarStr.Data());
+
     fHistMan = new HistogramManager("analysisHistos", "aa", VarManager::kNVars);
     fHistMan->SetUseDefaultVariableNames(kTRUE);
     fHistMan->SetDefaultVarNames(VarManager::fgVariableNames, VarManager::fgVariableUnits);
 
     TString histNames = "";
-    histNames += "PairsMuonSEPM;PairsMuonSEPP;PairsMuonSEMM;";
+    std::vector<float> fCentLimsHashing = fMixHandler->GetMixingVariableLimits("Cent");
+    for (int icent = 0; icent < fCentLimsHashing.size() - 1; ++icent) {
+      fCentBinNames.push_back(Form("%.1f%.1f", fCentLimsHashing[icent], fCentLimsHashing[icent + 1]));
+      histNames += Form("PairsMuonSEPM_%s;PairsMuonSEPP_%s;PairsMuonSEMM_%s;", fCentBinNames[icent].Data(), fCentBinNames[icent].Data(), fCentBinNames[icent].Data());
+      //        std::cout<<"  CentralityBin "<< icent<<" : "<< fCentLimsHashing[icent]<<"-"<<fCentLimsHashing[icent + 1]<<std::endl;
+    }
```

Comment:
As i commented above, you do not need more than one mixed event histogram directory.

Reply:
See above